packages/repository/src/relations/has-many/has-many-through.repository.ts

```diff
@@ -98,3 +105,129 @@ export interface HasManyThroughRepository<
     },
   ): Promise<void>;
 }
+
+export class DefaultHasManyThroughRepository<
```

Comment:
Let's add API docs for it.

Reply:
+1, also include "experimental feature" warning.